### PR TITLE
Redoing html with Materialize-css

### PR DESCRIPTION
**Pull request by nobobo1234**
This improves UX with a responsive interface. It might go a bit against what I told you in the earlier pr but this is a tad easier to make. You might consider using something like pug to improve the readability of your html but then again this was done pretty fast.

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -1,219 +1,107 @@
...
-			>licensed under AGPLv3</a
-		>
-	</footer>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dream Simulator</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (aspenluxxxy) [inline web/index.html:6]: Going to self-host the fonts/css on my site

**Review by aspenluxxxy — APPROVED** [review]
Thank you!

File: web/index.html (hunk 1 above)
Comment (nobobo1234) [reply web/index.html:6]: Sure thing.

**Comment by aspenluxxxy** [comment]
Also, the `blaze.png:Zone.Identifier` prevents cloning on Windows.

**Comment by nobobo1234** [comment]
Never knew I committed those. Do I need to remove them?

**Comment by aspenluxxxy** [comment]
i got rid of em

[here's an explanation of where they came from](https://apple.stackexchange.com/questions/378438/what-are-these-extra-zone-identifier-files-created-during-windows-remote-deskto)

**Pull request merged**